pkg/acsengine/engine.go

```diff
@@ -758,6 +758,44 @@ func (t *TemplateGenerator) getTemplateFuncMap(cs *api.ContainerService) map[str
 		"HasWindowsSecrets": func() bool {
 			return cs.Properties.WindowsProfile.HasSecrets()
 		},
+		"PopulateClassicModeDefaultValue": func(attr string) string {
+			var val string
+			if !t.ClassicMode {
+				val = ""
```

Comment:
"" defaultValue was there for a few parameters. 
And yes, there are places doesn't have defaultValue set at all.

So i want to keep it this way. It is a more common case.